openstack/clustering/v1/clusters/results.go

```diff
@@ -0,0 +1,127 @@
+package clusters
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"reflect"
```

Comment:
nit: have reflect bundled with the other standard library packages.

Reply:
Done